wolfcrypt/src/pkcs12.c

```diff
@@ -661,6 +695,39 @@ int wc_d2i_PKCS12(const byte* der, word32 derSz, WC_PKCS12* pkcs12)
         return ret;
     }
 
+    #ifdef ASN_BER_TO_DER
+     if (size == 0) {
```

Comment:
What are we seeing indefinite length here if we converted to DER above?

Reply:
This code is executed first and is the first call to BerToDer. The second call is for indefinite length encrypted data.